jobs/mongodb_backup_job.go

```diff
@@ -0,0 +1,261 @@
+// pmm-agent
+// Copyright 2019 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jobs
+
+import (
+	"context"
+	"io/ioutil"
+	"net"
+	"net/url"
+	"os"
+	"os/exec"
+	"regexp"
+	"strconv"
+	"time"
+
+	"github.com/golang/protobuf/ptypes"
+	"github.com/percona/pmm/api/agentpb"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	pbmBin = "pbm"
```

Comment:
it would be good to make path configurable via `pmm-agent.yaml` file

Reply:
Well, for now we expect that `pbm` is available in `PATH`. Later we will ship this binary with `pmm-agent`. If we want this configuration option now better to create separate ticket, because we already have hardcoded binaries for MySQL backup/restore.